pkg/skaffold/deploy/kpt/kpt.go

```diff
@@ -312,7 +312,28 @@ func (k *Deployer) renderManifests(ctx context.Context, _ io.Writer, builds []bu
 		if err != nil {
 			return nil, fmt.Errorf("kustomize build: %w", err)
 		}
+	} else {
```

Comment:
I would add a comment here to just explain why this workaround is needed because I don't think it's super clear what's happening here without context from the github issue/PR